tools/mo/openvino/tools/mo/back/offline_transformations.py

```diff
@@ -88,9 +88,9 @@ def apply_user_transformations(func: object, transforms: list):
         available_transformations[name](func, **args)
 
 
-def apply_moc_transformations(func: object):
+def apply_moc_transformations(func: object, static_shape):
```

Comment:
```suggestion
def apply_moc_transformations(func: object, static_shape: bool):
```

Reply:
Does anyone use this particular function? I mean POT or other tools since we are changing signature of this function?

Reply:
Yes, they are most likely used by optimum-intel. Should we create a new method with new signature?

Reply:
Used `apply_moc_transformations()` directly from `openvino._offline_transformations`, so now we don't need to change this method.